osu.Game/Screens/Play/Player.cs

```diff
@@ -345,6 +345,11 @@ public override void OnEntering(IScreen last)
                 .Delay(250)
                 .FadeIn(250);
 
+            ShowStoryboard.ValueChanged += _ => UpdateBackgroundElements();
```

Comment:
is this actually required? i mean the part where it's calling the base logic. it doesn't feel right since i'm pretty sure you already have bindables taking care of this flow.

Reply:
Moving the bindable update triggered storyboard initialization here to resolve this.